mava/components/jax/building/data_server.py

```diff
@@ -99,12 +99,155 @@ def name() -> str:
         return "data_server"
 
 
+@dataclass
+class SamplerConfig:
+    pass
+
+
+class Sampler(Component):
```

Comment:
I'm inclined to say that these two new components and the `rate_limiters` should live in a file `components/jax/building/reverb.py`, or something along those lines.

Reply:
Done - moved it into a file called reverb.py